messages/src/did_doc/service_aries_public.rs

```diff
@@ -0,0 +1,85 @@
+pub const SERVICE_SUFFIX: &str = "indy";
+pub const SERVICE_TYPE: &str = "IndyAgent";
+
+// Service object as defined https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md#the-services-item
+// Note that is divergence from w3c spec https://w3c.github.io/did-core/#service-properties
+#[derive(Serialize, Deserialize)]
+pub struct EndpointService {
+    pub endpoint: String,
+    #[serde(rename = "recipientKeys")]
+    pub recipient_keys: Option<Vec<String>>,
```

Comment:
To be aligned with the spec, let's also remove `recipientKeys` and focus strictly on the Stage1 of https://github.com/hyperledger/aries-vcx/issues/609 and reflecting did:sov method endpoint ATTRIB structure here 1:1